web/package.json

```diff
@@ -97,5 +97,9 @@
     "karma-notify-reporter": "^0.1.1",
     "karma-sourcemap-loader": "^0.3.5",
     "karma-webpack": "^1.7.0"
+  },
+  "devDependencies": {
+    "nodemon": "^1.7.0",
+    "webpack-dev-server": "^1.11.0"
```

Comment:
There's actually an issue for that #430. If you have enough time to separate them between dev and prod ones, feel free to update!